mill

```diff
@@ -20,9 +20,13 @@ fi
 MILL_DOWNLOAD_PATH="$HOME/.mill/download"
 MILL_EXEC_PATH="${MILL_DOWNLOAD_PATH}/$MILL_VERSION"
 
+version_remainder="$MILL_VERSION"
+MILL_MAJOR_VERSION="${version_remainder%%.*}"; version_remainder="${version_remainder#*.}"
```

Comment:
Just to be sure, is this `sh` compatible or is this `bash` specific? It looks like it will only work with bash.

Reply:
I think it's sh compatible (at least intellij shell plugin isn't giving me any POSIX warnings and it seems to run OK on bash in POSIX mode).
Substring removal is used here as well https://github.com/lihaoyi/mill/blob/master/mill#L26 so we'll need to change that if string manipulation is not compatible with our target.
Are we targeting POSIX shells or another?

Reply:
Yep, POSIX shells. Thanks for checking.